test/plugin/test_out_google_cloud.rb

```diff
@@ -764,8 +763,10 @@ def test_timestamps
     verify_log_entries(emit_index, COMPUTE_PARAMS) do |entry|
       assert_equal expected_ts[verify_index].tv_sec,
                    entry['timestamp']['seconds'], entry
-      assert_equal expected_ts[verify_index].tv_nsec,
-                   entry['timestamp']['nanos'], entry
+      # Fluentd v0.14 onwards supports nanosecond timestamp values.
+      # Added in 1 ms delta to avoid flaky tests.
```

Comment:
I would add the above explanation here and reference [the Wikipedia article](https://en.wikipedia.org/wiki/Double-precision_floating-point_format).